packages/react-bindings/src/hooks/useAccessibility.ts

```diff
@@ -0,0 +1,67 @@
+import { Accessibility, AccessibilityAttributesBySlot } from '@stardust-ui/accessibility'
+import * as React from 'react'
+
+import getAccessibility from '../accessibility/getAccessibility'
+import { AccessibilityBehavior, AccessibilityActionHandlers } from '../accessibility/types'
+
+type UseAccessibilityOptions<Props> = {
+  actionHandlers?: AccessibilityActionHandlers
+  debugName?: string
+  mapPropsToBehavior?: () => Props
```

Comment:
can this be `Props | () => Props` for API simplification? looking at the unit tests I *think* passing an object there should be sufficient for most use cases.

Reply:
I see that you mean, but I want to keep it obvious for now and in sync with `useStateManager()`. We can improve this part once we will receive feedback for this thing.

Reply:
I agree that `Props | () => Props` maybe make sense, in most of the cases I see this as a simple props mapping.. Anyways, not a blocker :)